### PR TITLE
V5 ga 80 custom response editor

### PR DESCRIPTION
**Pull request by SrdjanReblaze**
image
https://reblaze.atlassian.net/jira/software/projects/V5GA/boards/39?selectedIssue=V5GA-80

### CODE REVIEW

File: src/views/security/CustomResponsesEditor.vue [hunk 1]
```diff
@@ -88,16 +68,19 @@
...
-                         placeholder="Header value"
-                         v-model="header.value"/>
+                  <form-input name="document-header-value"
+                              classes="document-header-value"
+                              title="Header value"
+                              placeholder="Header value"
+                              v-model:inputValue="header.value"/>
                 </div>
                 <div class="column is-narrow">
                   <button class="button is-light is-small remove-icon is-small has-text-grey"
```
Comment (Aviv-Galmidi) [inline src/views/security/CustomResponsesEditor.vue:86]: Button was not converted

File: src/views/security/CustomResponsesEditor.vue [hunk 2]
```diff
@@ -109,7 +92,7 @@
               </div>
               <a title="Add new header"
                  class="is-text is-small is-size-7 ml-3 add-key-button"
-                 data-qa="add-new-key-btn"
+                 dataQa="add-new-key-btn"
```
Comment (Aviv-Galmidi) [inline src/views/security/CustomResponsesEditor.vue:95]: `data-qa` is the correct format, attributes are declared as camelCase in the component but used in kebab-case when implemented in HTML
Please change in the rest of the code added in this PR

**Review by Aviv-Galmidi — CHANGES_REQUESTED** [review]

File: src/views/security/CustomResponsesEditor.vue (hunk 1 above)
Comment (SrdjanReblaze) [reply src/views/security/CustomResponsesEditor.vue:86]: done

File: src/views/security/CustomResponsesEditor.vue (hunk 2 above)
Comment (SrdjanReblaze) [reply src/views/security/CustomResponsesEditor.vue:95]: done

**Review by Aviv-Galmidi — CHANGES_REQUESTED** [review]
1. Header title is duplicated for each entry of a header, we don't need this title at all as we have the `Request Headers` above it
new
Screen Shot 2023-05-18 at 13 13 44
old
Screen Shot 2023-05-18 at 13 13 35

**Comment by SrdjanReblaze** [comment]
> 1. Header title is duplicated for each entry of a header, we don't need this title at all as we have the `Request Headers` above it
>    new
>    Screen Shot 2023-05-18 at 13 13 44
>    old
>    Screen Shot 2023-05-18 at 13 13 35

done

**Review by Aviv-Galmidi — APPROVED** [review]

**Pull request merged**